### PR TITLE
ENH Update keep me signed in tooltip text

### PR DESCRIPTION
**Pull request by emteknetnz**
https://github.com/silverstripe/silverstripe-login-forms/issues/90

### CODE REVIEW

File: lang/en.yml [hunk 1]
```diff
@@ -267,8 +267,8 @@ en:
     FIRSTNAME: 'First Name'
     INTERFACELANG: 'Interface Language'
     KEEPMESIGNEDIN: 'Keep me signed in'
-    REMEMBER_ME: 'Remember me for {count} days'
-    KEEP_ME_SIGNED_IN_TOOLTIP: 'You will remain authenticated on this device for {count} days. Only use this feature if you trust the device you are using. Authenticated devices can be managed in your profile.'
+    REMEMBER_ME: 'Keep me signed in for {count} days'
```
Comment (maxime-rainville) [inline lang/en.yml:270]: I thought we said we weren't going to update translation string any more?

**Review by maxime-rainville — CHANGES_REQUESTED** [review]
Not clear why we are targeting this fix to 4.7.

File: lang/en.yml (hunk 1 above)
Comment (emteknetnz) [reply lang/en.yml:270]: Ug. Er, I don't think that was agreed. 

Last I got to was here - https://github.com/silverstripeltd/product-issues/issues/361#issuecomment-816279940

I think our process should probably be flexible enough to allow us to update text :-)

Leaving transifix woes aside, this text change is only a tweak which means roughly the same thing.  It's not repurposing an existing key for something else, so I think it's correct?  Means we don't automatically loose foreign translations.  So it seems correct to use the same key?

**Comment by emteknetnz** [comment]
>> Not clear why we are targeting this fix to 4.7.

There's no API change, so it can be patch released

File: lang/en.yml (hunk 1 above)
Comment (maxime-rainville) [reply lang/en.yml:270]: If you update an existing translation string, it invalidate all translation attach to that string ... even if you keep the same key.

Comment (emteknetnz) [reply lang/en.yml:270]: ノ(;Ĺ̯̿̿ ;ノ)

**Comment by maxime-rainville** [comment]
Patch releases are normally bugfixes only. So it goes a bit beyond no new API.

We haven't interpreted it to be a hard rule in the past, so I'm OKish with tweaking translations. But generally people should't expect to see new UIs and new words in a patch release.

File: lang/en.yml (hunk 1 above)
Comment (emteknetnz) [reply lang/en.yml:270]: In this particularly instance, the modified translation keys haven't actually been released / tx pushed yet, so it's OK to modify in this particular PRs

**Comment by brynwhyman** [comment]
If it's possible to treat a UI change as a bug fix, then this is a bug fix. As per the parent issue title, changes that were previously introduced to 4.7 included "Inaccurate help text".

**Pull request merged**